map_server/srv/LoadMap.srv

```diff
@@ -0,0 +1,3 @@
+string map_path
+---
+bool success
```

Comment:
I'd really rather we not put messages in the packages in navigation --  we have a separate repo (https://github.com/ros-planning/navigation_msgs) that is for messages, this has the advantage of making the interface itself (the messages) lightweight to build, especially if someone is on an alternate platform where they can't actually build/install the full navigation stack, but they just want to communicate with it